doajtest/testbook/dashboard/historical_stats.yml

```diff
@@ -0,0 +1,97 @@
+  - title: Historical Statistics Display - Editor with 2 groups
+    context:
+      role: editor_0
+      testdrive: statistics
+    steps:
+      - step: Navigate to "/testdrive/statistics" and wait for the page to load slowly. This should be done once at the beginning of the test session and not refreshed between individual tests.
+      - step: Log in with provided editor_0 credentials.
+        results:
+          - User dashboard is displayed.
+      - step: Verify the motivational banner at the top of the dashboard (green) and check number of finished applications.
+        results:
+          - The motivational banner is GREEN and shows the correct sum of applications finished in both groups.
+      - step: Verify historical statistics in the Activity section at the bottom.
+        results:
+          - Two groups are displayed.
+          - Only associate editors are shown in the Activity section for each group.
+          - Each group has the correct statistics for the current year.
+          - The number of READY/COMPLETED applications matches the "additional information" table.
+      - step: Log out.
+      - step: (Optional) Click the link at the bottom of the testdrive page to free resources after finishing this test session.
+  - title: Historical Statistics Display - Editor with 1 group
+    context:
+      role: editor_1
+      testdrive: statistics
+    steps:
+      - step: Navigate to "/testdrive/statistics" and wait for the page to load slowly. This should be done once at the beginning of the test session and not refreshed between individual tests.
+      - step: Log in with provided editor_1 credentials.
+        results:
+          - User dashboard is displayed.
+      - step: Verify the motivational banner at the top of the dashboard (green) and check number of finished applications.
+        results:
+          - The motivational banner is GREEN and shows the correct sum of applications finished in the group.
+      - step: Verify historical statistics in the Activity section at the bottom.
+        results:
+          - One group is displayed.
+          - Only associate editors are shown in the Activity section for the group.
+          - The group has the correct statistics for the current year.
+          - The number of READY/COMPLETED applications matches the "additional information" table.
+      - step: Log out.
+      - step: (Optional) Click the link at the bottom of the testdrive page to free resources after finishing this test session.
+  - title: Historical Statistics Display - Associate Editors with 0 finished applications
+    context:
+      role: associate editor
+      testdrive: statistics
+    steps:
+      - step: Navigate to "/testdrive/statistics" and wait for the page to load slowly. This should be done once at the beginning of the test session and not refreshed between individual tests.
+      - step: Log in with provided associate_editor_0 credentials.
```

Comment:
There is no associate_editor_0 credentials, I think this is just an off-by-one numbering issue, as there is _1 and _2 instead